README.md

```diff
@@ -45,6 +45,15 @@ So if you want to get all the examples, make sure to clone the repo recursively
 
   This model powers the [Move One iPhone app](https://apps.apple.com/us/app/move-one/id6448635527)
 
+### multicam
+
+* Full body tracking
+* Hand and finger tracking
+* FBX, USD and Blend 3D output formats
+* Supports any video but best results are obtained when the camera is known
+
```

Comment:
@benmoveai - are there other bullet points which might be missing here?

Reply:
I'm the wrong person to ask. Ross or Jack are the best ones for this

Reply:
@benmoveai - This has been updated now